.github/actions/test/action.yml

```diff
@@ -0,0 +1,49 @@
+# https://docs.github.com/en/actions/creating-actions/creating-a-composite-action
```

Comment:
I think we can drop the `.github/actions/test/action.yml` and just let flowzone run on all node versions it supports.